src/uct/api/v2/uct_v2.h

```diff
@@ -223,6 +276,26 @@ uct_iface_estimate_perf(uct_iface_h tl_iface, uct_perf_attr_t *perf_attr);
 ucs_status_t uct_md_mem_dereg_v2(uct_md_h md,
                                  const uct_md_mem_dereg_params_t *params);
 
+
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Check if remote iface address is reachable.
+ *
+ * This function checks if a remote address can be reached from a local
+ * interface. If the function returns a non-zero value, it does not necessarily
+ * mean a connection and/or data transfer would succeed, since the reachability
+ * check is a local operation it does not detect issues such as network
```

Comment:
"succeed, since" -> "succeed; as the ..."

Reply:
fixed